docs/yaml-config.rst

```diff
@@ -64,7 +64,7 @@ The path to your pip requirements file.
 
 .. code-block:: yaml
 
-   requirements_file: requirements/docs.txt
+   requirements_file: docs/requirements.txt
```

Comment:
Actually, this was on purpose, some projects have all the requirements in a directory called `requirements` (we do), but yeah the `docs/requirements.txt` pattern is used too.

Reply:
Ah well if this is on purpose, then I should close this, sorry for the noise.

Reply:
Don't worry